<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QDialog" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>517</width>
    <height>409</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="verticalSpacing">
      <number>6</number>
     </property>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Slots</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="SpinBoxSlots">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>256</number>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QSpinBox" name="SpinBoxGridY">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>256</number>
       </property>
      </widget>
     </item>
     <item row="6" column="3">
      <widget class="QSpinBox" name="SpinBoxYearsPerIteration">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>99</number>
       </property>
       <property name="value">
        <number>15</number>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>Years/Iteration</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QSpinBox" name="SpinBoxEnvChangeRate">
       <property name="maximum">
        <number>100000</number>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>Env. change rate</string>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QSpinBox" name="SpinBoxDifferenceBreed">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>31</number>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Max Diff to Breed</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QSpinBox" name="SpinBoxMutationChance">
       <property name="maximum">
        <number>255</number>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Chance of Mutation</string>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QSpinBox" name="SpinBoxBreedCost">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>10000</number>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Breed Cost</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QSpinBox" name="SpinBoxFood">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>20000</number>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Energy Input</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QSpinBox" name="SpinBoxDispersal">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>200</number>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Dispersal</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="SpinBoxStartAge">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Start Age</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QSpinBox" name="SpinBoxBreedThreshold">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>5000</number>
       </property>
       <property name="value">
        <number>250</number>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Breed Threshold</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="SpinBoxSettleTolerance">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>30</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Settle tolerance</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QSpinBox" name="SpinBoxFitnessTarget">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>96</number>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Fitness target</string>
       </property>
      </widget>
     </item>
     <item row="11" column="2">
      <widget class="QLabel" name="label_22">
       <property name="text">
        <string>Use Max Diff to Breed</string>
       </property>
      </widget>
     </item>
     <item row="11" column="1" alignment="Qt::AlignHCenter">
      <widget class="QCheckBox" name="CheckBoxBreedWithinSpecies">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="9" column="3" alignment="Qt::AlignHCenter">
      <widget class="QCheckBox" name="CheckBoxToroidal">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="9" column="2">
      <widget class="QLabel" name="label_19">
       <property name="text">
        <string>Toroidal</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1" alignment="Qt::AlignHCenter">
      <widget class="QCheckBox" name="CheckBoxNonSpatial">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label_20">
       <property name="text">
        <string>Nonspatial Settling</string>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="label_21">
       <property name="text">
        <string>Breed only within species</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Grid Y</string>
       </property>
      </widget>
     </item>
     <item row="11" column="3" alignment="Qt::AlignHCenter">
      <widget class="QCheckBox" name="CheckBoxBreedWithDifference">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="SpinBoxGridX">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>256</number>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Grid X</string>
       </property>
      </widget>
     </item>
     <item row="13" column="1">
      <widget class="QRadioButton" name="radioSexual">
       <property name="text">
        <string>Sexual</string>
       </property>
      </widget>
     </item>
     <item row="13" column="2" alignment="Qt::AlignHCenter">
      <widget class="QRadioButton" name="radioAsexual">
       <property name="text">
        <string>Asexual</string>
       </property>
      </widget>
     </item>
     <item row="13" column="3">
      <widget class="QRadioButton" name="radioVariable">
       <property name="text">
        <string>Variable</string>
       </property>
      </widget>
     </item>
     <item row="12" column="1" alignment="Qt::AlignHCenter">
      <widget class="QCheckBox" name="CheckBoxRecalcFitness">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="13" column="0">
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>Breeding</string>
       </property>
      </widget>
     </item>
     <item row="12" column="0">
      <widget class="QLabel" name="lableRF">
       <property name="text">
        <string>Recalc Fitnesses</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Settings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Settings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
